test_results.proto

```diff
@@ -26,7 +26,6 @@ message TestResult {
   string test_name = 1;
   int32 trial = 2;
   string status = 3;
-  string log_url = 4;
```

Comment:
Fix the numbers here, we are now skipping 4.

Reply:
Done.